README.md

```diff
@@ -57,4 +57,10 @@ https://www.dealii.org.
 Continuous Integration Status:
 ------------------------
 
-[![Build Status](https://travis-ci.org/dealii/dealii.png)](https://travis-ci.org/dealii/dealii)
+| System | Status | Info |
+| --- | --- | --- |
+| indent | [![Build Status](https://travis-ci.org/dealii/dealii.png)](https://travis-ci.org/dealii/dealii) | using https://travis-ci.org |
+| Linux | https://jenkins.tjhei.info/job/dealii/ | build on https://jenkins.tjhei.info |
```

Comment:
I would probably just get rid of this line and say in `CDash` "various Linux builds".

Reply:
There is a difference between the jenkins CI and CDash, so why would you want to merge them? Or maybe I am misunderstanding?

Reply:
When I click on https://jenkins.tjhei.info/job/dealii/, I don't really see anything meaningful regarding the current build status (or are you going to add something there such that it looks similar to https://jenkins.tjhei.info/job/dealii/view/change-requests/?). I would argue that currently all the CI testing for `master` on `Linux` systems happens on `CDash`. Hence, I would only mention that.